docs/changelog/2024.md

```diff
@@ -64,6 +81,50 @@ More changes:
 - 🔥 Remove `.list_source()` [PR](https://github.com/laminlabs/bionty/pull/130) [@sunnyosun](https://github.com/sunnyosun)
 - 🎨 Remove update in add_ontology_from_df [PR](https://github.com/laminlabs/bionty/pull/129) [@sunnyosun](https://github.com/sunnyosun)
 
+## 2024-10-14 {small}`hub 0.29.0`
+
+### Enhancements
+
+#### Improve filtering
```

Comment:
Definitely not 4 levels of headings unless we put extensive effort into styling it.

Not in 2024.

Reply:
Fixed.